src/index.js

```diff
@@ -61,11 +64,20 @@ const formatOptions = ({steps, ...otherOptions}) => ({
   steps: (steps || ['welcome','document','face','complete']).map(formatStep)
 })
 
+const setI18n = (options) => {
+  const locale = options.language ? options.language.locale : 'en'
+  const customStrings = options.language ? options.language.customStrings : null
+  const phrases = locales[locale]
+  const polyglot = new Polyglot({locale, phrases})
+  if (customStrings) polyglot.extend(customStrings)
+  return polyglot
+}
 
 Onfido.init = (opts) => {
   console.log("onfido_sdk_version", process.env.SDK_VERSION)
   Tracker.track()
-  const options = formatOptions({ ...defaults, ...opts, events })
+  const i18n = setI18n(opts)
+  const options = formatOptions({ ...defaults, ...opts, events, i18n })
```

Comment:
Do we need to pass it down to all components? Would it work as a global utility function?

Reply:
Yes, I could use a function to utility function to initialise polyglot and a HOC to wrap up all the components with the translations.

Reply:
I was reading [this article](https://www.jonathan-petitcolas.com/2017/03/27/internationalizing-react-application-using-polyglot.html) that tackles this probles using HOC and context. It seems like a good way to fix this issue. I would do it as part of another story though.

Reply:
Interesting article. What if we just use a global?

Reply:
Uhm, I am a bit wary of using a global function, seems like code smell to me because 2 globals with the same name might break the code and it seems safer to use this solution. What are your thoughts @rfreitas?

Reply:
Unless you mean that this code could be extracted into a utils function, but that would mean that we would have to instantiate `polyglot` every time we import it.

Reply:
Could we have a global `i18n` object in utils? Then we can just import it and call `i18n.t` whenever we need a string?

I'm not sure how you would instantiate that object without using the window, but it would make passing `i18n` around all over the place go away.

Regardless I think we should do some thinking to avoid passing it around. I think we should do it as part of this PR because adding `i18n` to all the props is probably unnecessary.